scripts/monitoring/cron-send-user-running-pod-count.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+
+COUNT=$(oadm manage-node --list-pods --selector=type=compute | grep Running | wc -l)
+echo
+echo "Number of User Running pod account : $COUNT"
```

Comment:
Please disregard this comment. With ops-runner, we'll handle the output. We're not using cron e-mails to detect failures in v3, we're using zabbix instead.